trailblazer/store/crud/update.py

```diff
@@ -179,10 +179,11 @@ def cancel_ongoing_analysis(
         LOG.info(f"Case {analysis.case_id} - Analysis {analysis.id}: cancelled successfully!")
         self.update_run_status(analysis_id=analysis_id, analysis_host=analysis_host)
         analysis.status = TrailblazerStatus.CANCELLED
-        analysis.comment = (
+        new_comment: str = (
             f"Analysis cancelled manually by user:"
             f" {(self.get_user(email=email).name if self.get_user(email=email) else (email or 'Unknown'))}!"
```

Comment:
Could adding who be done in a function?

Reply:
Refactored another method slightly so it could be reused for this purpose now.